library/std/src/sys_common/net/tests.rs

```diff
@@ -6,7 +6,7 @@ fn no_lookup_host_duplicates() {
     let mut addrs = HashMap::new();
     let lh = match LookupHost::try_from(("localhost", 0)) {
         Ok(lh) => lh,
-        Err(e) => panic!("couldn't resolve `localhost': {e}"),
+        Err(e) => panic!("couldn't resolve localhost: {e}"),
```

Comment:
I think the original intent here is to have some backticks, because we do mean literally the string "localhost" not the abstract idea.